### ISSUE TITLE
prévoir un espace pour lister les crédits (institutions et personnes individuelles)

### ISSUE THREAD

**Issue by EliCommins** [davidonet/Textopoly#47]

**Comment by davidonet** [comment]
Moi je verrais bien des billets pour ça...
Avec un bouton dans l'interface qui nous y amène

**Comment by EliCommins** [comment]
non, j'ai besoin d'un niveau de "para-Textopoly", ,notamment pour indiquer
les crédits. dans le storyboard cela prend la forme du bandeau supérieur
qui descend, en symétrie avec la boîte qui apparaît par le bas de la page.

2012/10/1 David Olivari notifications@github.com
